Sources/SafeDITool/SafeDITool.swift

```diff
@@ -23,7 +23,7 @@ import Foundation
 import SafeDICore
 import SwiftParser
 #if canImport(ZippyJSON)
-    @preconcurrency import ZippyJSON
```

Comment:
honestly feels like a bug that `ZippyJSONDecoder` isn't treated as Sendable given that I was doing a `@preconcurrency import` but... oh well.